src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1781,7 +1783,9 @@ def __setitem__(self, name: str, value):
         child.set_state(value)
 
 
-class _HasAllowedValuesMixin:
+class HasAllowedValuesMixin:
```

Comment:
because it's based on `has_allowed_values` flag in static info. https://github.com/ansys/pyfluent/pull/1597

![image](https://github.com/ansys/pyfluent/assets/106588300/2228054d-7dfe-4dff-9c22-566de0e288ac)

Reply:
Would it make more sense if it were just AllowedValuesMixin since that is what we're mixing in? If we decide on this name change, now is an opportune moment.

Reply:
Done. Thank you.

Reply:
Thanks! I will approve.